prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -447,10 +459,8 @@ def prompt_processor() -> Any:
                     If the context is trying to convey that the answer is true, \
                     then return "yes", else return "no".'
                 answer = run_completion(
-                    llm_helper=llm_helper,
-                    llm_li=llm_li,
-                    prompt=prompt,
-                    adapter_instance_id=adapter_instance_id,
+                    llm,
+                    prompt,
```

Comment:
Can we pass this as named args instead of positional args for better readability?

Reply:
Done